File: C#/백준/Bronze/32306. Basketball Score/Basketball Score.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = new int[2];
            for (int i=0;i<2;i++){
                int[] b = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                a[i] = b[0]+b[1]*2+b[2]*3;
            }
            if(a[0]==a[1]){
                Console.WriteLine(0);
            }
            else{
                Console.WriteLine(a[0]>a[1]?1:2);
            }
        }
    }
}

File: Python/백준/Bronze/32306. Basketball Score/Basketball Score.py
aa=[]
for i in range(2):
    a,b,c = map(int,input().split())
    aa.append(a+b*2+c*3)
if(aa[0]==aa[1]):
    print(0)
elif(aa[0]>aa[1]):
    print(1)
else:
    print(2)